{"name": "Successful login", "status": "broken", "statusDetails": {"message": "AttributeError: 'Funciones_Globales' object has no attribute 'dashboard'\n", "trace": "  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\steps_login.py\", line 53, in step_impl\n    f.dashboard(\"xpath\",\"//*[@id='root']/section/section/header/div/div[1]/div/img\",)\n"}, "steps": [{"name": "Given I am on the login page", "status": "passed", "start": 1679500880692, "stop": 1679500885308}, {"name": "When I enter my valid username and password", "status": "passed", "start": 1679500885309, "stop": 1679500886530}, {"name": "And I click on the login button", "status": "passed", "start": 1679500886531, "stop": 1679500887088}, {"name": "Then I should see my personal dashboard", "status": "broken", "statusDetails": {"message": "AttributeError: 'Funciones_Globales' object has no attribute 'dashboard'\n", "trace": "  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\repo\\selenium\\maquina_uno\\mv1\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\steps_login.py\", line 53, in step_impl\n    f.dashboard(\"xpath\",\"//*[@id='root']/section/section/header/div/div[1]/div/img\",)\n"}, "start": 1679500887088, "stop": 1679500887097}], "start": 1679500880692, "stop": 1679500887098, "uuid": "b04ec969-7a5e-42b6-a5d8-b8517d835283", "historyId": "82e6fd88a69b5968ce989053b7ddc5a8", "fullName": "Login: Successful login", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}